src/element-service.js

```diff
@@ -203,6 +203,33 @@ export function extensionScriptsInNode(head) {
   return scripts;
 }
 
+/**
+ * Waits for body to be present then verifies that an extension script is
+ * present in head for installation.
+ * @param {!Promise<!Element>} whenbodyAvailPromise
+ * @param {HTMLHeadElement|Element|ShadowRoot} head
+ * @param {string} extensionId
+ * @return {!Promise<boolean>}
+ */
+export function isExtensionScriptInNode(
+  whenbodyAvailPromise, head, extensionId) {
```

Comment:
Pass in the `ampdoc` instead of them providing the `whenBody` and `head`.

Reply:
done.